Gemfile

```diff
@@ -21,9 +21,8 @@ rails_version = ENV["RAILS_VERSION"] || "default"
 
 rails = case rails_version
   when "default"
-    ">= 3.1.0"
```

Comment:
we're still using `3.2.19` :( and `ENV["RAILS_VERSION"]` is `nil` unless we specifically set it for this gem. why enforce a Rails version at all? :palm_tree: